### PR TITLE
add style to copyinput

### PR DESCRIPTION
**Pull request by ming08108**
Adds some styling to the copy input.

image

### CODE REVIEW

**Comment by akhilcacharya** [comment]
The input select border looks strange, IMHO. Can we disable that or just not use an input?

**Comment by vapaavetehinen** [comment]
should we have some padding for the input box? seems like the text starts very close to the left edge on some of the letters.

File: src/CopyInput.svelte [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+  button {
+    border: none;
+    background-color: #008cba;
+    color: white;
+    padding: 0.5em;
+  }
+  input {
+    border: none;
+    width: 25em;
+  }
```
Comment (vapaavetehinen) [inline src/CopyInput.svelte]: would be good to add just a little bit of padding here.

**Comment by ming08108** [comment]
> The input select border looks strange, IMHO. Can we disable that or just not use an input?

We need the input select. But we can style it however we want.

**Review by vapaavetehinen — APPROVED** [review]

**Pull request merged**